src/js/tracks/text-track-display.js

```diff
@@ -266,7 +266,7 @@ class TextTrackDisplay extends Component {
    *        Text track object to be added to the list.
    */
   updateForTrack(track) {
-    if (typeof window.WebVTT !== 'function' || !track.activeCues) {
+    if (typeof window.WebVTT !== 'function' || !track.activeCues || this.player_.textTrackSettings === false) {
```

Comment:
instead of exiting this function entirely, what about moving the check to be around `overrides` below?
something like
```js
let overrides = {};
if (this.player_.textTrackSettings) {
  overrides = this.player_.textTrackSettings.getValues();
}
```
That way captions still load properly it just doesn't bother getting overrides.

Reply:
Actually what if we move overrides down after processCues that way that function is called, but then do the early out after that just to avoid the while loop. I mean nothing in the while loop will be run due to overrides being empty so why not skip it all together.

Reply:
That should be fine as well.

Reply:
what should I do in this case? close the pull request an make a new one? I dont see a way to update the commits of the current one.